.github/workflows/release-deploy-all.yml

```diff
@@ -88,7 +88,7 @@ env:
   QUEUE_PREFIX: ${{ secrets.QUEUE_PREFIX }}
   API_PORT: ${{ secrets.API_PORT }}
   WEB_PORT: ${{ secrets.WEB_PORT }}
-  DATABASE_NAME_KEY: ${{ secrets.DATABASE_NAME_KEY }}
+  DATABASE_NAME_PREFIX: ${{ secrets.DATABASE_NAME_PREFIX }}
```

Comment:
Do we really need the prefix? DATABASE_NAME could be something which is mapped from github env to deployment directly right?

Reply:
We can talk as a team and settle this. Maybe I am missing something but I was under the assumption that the DB name would go under the sims-env-creds maybe, but the current approach would also be acceptable.